packages/terra-status-view/src/StatusView.scss

```diff
@@ -0,0 +1,107 @@
+@import '~terra-mixins';
+
+:local {
+  .status-view {
+    height: 100%;
+    padding-bottom: var(--terra-status-view-padding-bottom, 0.714rem);
+    padding-top: var(--terra-status-view-padding-top, 0.714rem);
+  }
+
+  .status-view-inner {
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    width: 100%;
+  }
+
+  .glyph {
+    flex: 0 0 auto;
+    padding-bottom: 0.714rem;
+
+    > svg {
+      background-repeat: no-repeat;
+      background-size: contain;
+      height: 100px;
+      width: 100px;
+    }
+
+    > .no-data {
+      background: var(--terra-status-view-glyph-no-data, inline-svg("<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 48 48' class='is-bidi'><path fill='#868a8c' d='M45.8 12c1.2 0 2.2 1 2.2 2.2v26.7c0 1.2-1 2.2-2.2 2.2H2.2C1 43 0 42 0 40.8V7.2C0 6 1 5 2.2 5H15c1.4.1 2.7.6 3.7 1.6l3.8 3.9c1 1 2.3 1.5 3.7 1.6l19.4-.1h.2z'/></svg>"));
+    }
+
+    > .no-matching-results {
+      background: var(--terra-status-view-glyph-no-matching-results, inline-svg("<svg data-name='Layer 1' xmlns='http://www.w3.org/2000/svg' viewBox='0 0 48 48'><path fill='#868a8c' d='M47.4 43.2l-13-13A19.66 19.66 0 0 0 38 19a19 19 0 1 0-7.9 15.4l13 13a2.05 2.05 0 0 0 2.9 0l1.4-1.4a1.93 1.93 0 0 0 0-2.8zM3 19a16 16 0 1 1 16 16A16 16 0 0 1 3 19z'/></svg>"));
+    }
+
+    > .not-authorized {
+      background: var(--terra-status-view-glyph-not-authorized, inline-svg("<svg data-name='Layer 1' xmlns='http://www.w3.org/2000/svg' viewBox='0 0 48 48'><path fill='#868a8c' d='M38.8 20H35v-9a11 11 0 0 0-22 0v9H9.2A2.22 2.22 0 0 0 7 22.2v23.7A2.2 2.2 0 0 0 9.2 48h29.7a2.22 2.22 0 0 0 2.2-2.2V22.2a2.38 2.38 0 0 0-2.3-2.2zM26 33.5V40h-4v-6.5a3.26 3.26 0 0 1-1-2.3 3 3 0 0 1 3-3 3.08 3.08 0 0 1 3 3 3.26 3.26 0 0 1-1 2.3zm6-14.9V20H16v-9a8 8 0 0 1 16 0z'/></svg>"));
+    }
+
+    > .error {
+      background: var(--terra-status-view-glyph-error, inline-svg("<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 48 48'><path fill='#868a8c' d='M24 0c13.3 0 24 10.7 24 24S37.3 48 24 48 0 37.3 0 24C0 10.8 10.7 0 23.9 0h.1z'/><path fill='#FFF' d='M7 21.5h34v5H7v-5z'/></svg>"));
+    }
+
+    > .custom {
+      background: var(--terra-status-view-glyph-custom, none);
+    }
+  }
+
+  .message-content-group {
+    align-items: center;
+    display: block;
+    flex-direction: column;
+    padding-left: 0.714rem;
+    padding-right: 0.714rem;
+    width: 100%;
+  }
+
+  .title {
+    color: var(--terra-status-view-title-color, #868a8c);
+    flex: 0 0 auto;
+    flex-wrap: wrap;
+    font-family: var(--terra-status-view-title-font-family, 'HelveticaNeue');
+    font-size: var(--terra-status-view-title-font-size, 1.714rem);
+    text-align: center;
+    width: 100%;
+  }
+
+  .message {
+    color: var(--terra-status-view-message-color, #1c1f21);
+    flex: 0 0 auto;
+    flex-wrap: wrap;
+    font-family: var(--terra-status-view-message-font-family, 'HelveticaNeue');
```

Comment:
This property can be removed, font-family will cascade down to message from terra-base styles set on HTML element.